### PR TITLE
Add configuration of resourceexplorer2 (2), rolesanywhere (2), route53 (1), rum (2), s3 (1), sagemaker (2), scheduler (2), schemas (1), sesv2 (7), sqs (2), ssm (2), ssoadmin (3), transcribe (4), transfer (1), vpc_network (1) groups

### LINKED ISSUE #474: Configure resourceexplorer2 (2), rolesanywhere (2), route53 (1), rum (2), s3 (1), sagemaker (2), scheduler (2), schemas (1),  sesv2 (7), sqs (2), ssm (2), ssoadmin (3), transcribe (4), transfer (1), vpc_network (1)
Configure external name without testing:

* `resourceexplorer2`
  * [ ] [aws_resourceexplorer2_index](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/resourceexplorer2_index)
  * [ ] [aws_resourceexplorer2_view](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/resourceexplorer2_view)

* `rolesanywhere`
  * [ ] [aws_rolesanywhere_profile](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rolesanywhere_profile)
  * [ ] [aws_rolesanywhere_trust_anchor](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rolesanywhere_trust_anchor)

* `route53`
  * [ ] [aws_route53_resolver_config](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_resolver_config)

* `rum`
  * [ ] [aws_rum_app_monitor](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_app_monitor)
  * [ ] [aws_rum_metrics_destination](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_metrics_destination)

* `s3`
  * [ ] [aws_s3control_storage_lens_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3control_storage_lens_configuration)

* `sagemaker`
  * [ ] [aws_sagemaker_servicecatalog_portfolio_status](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_servicecatalog_portfolio_status)
  * [ ] [aws_sagemaker_space](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_space)

* `scheduler`
  * [ ] [aws_scheduler_schedule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule)
  * [ ] [aws_scheduler_schedule_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule_group)

* `schemas`
  * [ ] [aws_schemas_registry_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/schemas_registry_policy)

* `sesv2`
  * [ ] [aws_sesv2_configuration_set](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_configuration_set)
  * [ ] [aws_sesv2_configuration_set_event_destination](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_configuration_set_event_destination)
  * [ ] [aws_sesv2_dedicated_ip_assignment](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_dedicated_ip_assignment)
  * [ ] [aws_sesv2_dedicated_ip_pool](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_dedicated_ip_pool)
  * [ ] [aws_sesv2_email_identity](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_email_identity)
  * [ ] [aws_sesv2_email_identity_feedback_attributes](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_email_identity_feedback_attributes)
  * [ ] [aws_sesv2_email_identity_mail_from_attributes](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_email_identity_mail_from_attributes)

* `sqs`
  * [ ] [aws_sqs_queue_redrive_allow_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_allow_policy)
  * [ ] [aws_sqs_queue_redrive_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_policy)

* `ssm`
  * [ ] [aws_ssm_default_patch_baseline](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_default_patch_baseline)
  * [ ] [aws_ssm_service_setting](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_service_setting)

* `ssoadmin`
  * [ ] [aws_ssoadmin_customer_managed_policy_attachment](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssoadmin_customer_managed_policy_attachment)
  * [ ] [aws_ssoadmin_instance_access_control_attributes](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssoadmin_instance_access_control_attributes)
  * [ ] [aws_ssoadmin_permissions_boundary_attachment](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssoadmin_permissions_boundary_attachment)

* `transcribe`
  * [ ] [aws_transcribe_language_model](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_language_model)
  * [ ] [aws_transcribe_medical_vocabulary](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_medical_vocabulary)
  * [ ] [aws_transcribe_vocabulary](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary)
  * [ ] [aws_transcribe_vocabulary_filter](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary_filter)

* `transfer`
  * [ ] [aws_transfer_tag](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transfer_tag)

* `vpc_network`
  * [ ] [aws_vpc_network_performance_metric_subscription](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_network_performance_metric_subscription)

### PR DESCRIPTION
**Pull request by dverveiko**
### Description of your changes
Add external name configuration without testing:

* `resourceexplorer2`
  * [x] [aws_resourceexplorer2_index](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/resourceexplorer2_index)
  * [x] [aws_resourceexplorer2_view](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/resourceexplorer2_view)

* `rolesanywhere`
  * [x] [aws_rolesanywhere_profile](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rolesanywhere_profile)
  * [x] [aws_rolesanywhere_trust_anchor](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rolesanywhere_trust_anchor)

* `route53`
  * [x] [aws_route53_resolver_config](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_resolver_config)

* `rum`
  * [x] [aws_rum_app_monitor](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_app_monitor)
  * [x] [aws_rum_metrics_destination](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rum_metrics_destination)

* `s3`
  * [x] [aws_s3control_storage_lens_configuration](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3control_storage_lens_configuration)

* `sagemaker`
  * [x] [aws_sagemaker_servicecatalog_portfolio_status](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_servicecatalog_portfolio_status)
  * [x] [aws_sagemaker_space](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sagemaker_space)

* `scheduler`
  * [x] [aws_scheduler_schedule](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule)
  * [x] [aws_scheduler_schedule_group](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/scheduler_schedule_group)

* `schemas`
  * [x] [aws_schemas_registry_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/schemas_registry_policy)

* `sesv2`
  * [x] [aws_sesv2_configuration_set](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_configuration_set)
  * [x] [aws_sesv2_configuration_set_event_destination](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_configuration_set_event_destination)
  * [x] [aws_sesv2_dedicated_ip_assignment](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_dedicated_ip_assignment)
  * [x] [aws_sesv2_dedicated_ip_pool](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_dedicated_ip_pool)
  * [x] [aws_sesv2_email_identity](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_email_identity)
  * [x] [aws_sesv2_email_identity_feedback_attributes](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_email_identity_feedback_attributes)
  * [x] [aws_sesv2_email_identity_mail_from_attributes](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sesv2_email_identity_mail_from_attributes)

* `sqs`
  * [x] [aws_sqs_queue_redrive_allow_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_allow_policy)
  * [x] [aws_sqs_queue_redrive_policy](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/sqs_queue_redrive_policy)

* `ssm`
  * [x] [aws_ssm_default_patch_baseline](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_default_patch_baseline)
  * [x] [aws_ssm_service_setting](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_service_setting)

* `ssoadmin`
  * [x] [aws_ssoadmin_customer_managed_policy_attachment](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssoadmin_customer_managed_policy_attachment)
  * [x] [aws_ssoadmin_instance_access_control_attributes](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssoadmin_instance_access_control_attributes)
  * [x] [aws_ssoadmin_permissions_boundary_attachment](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssoadmin_permissions_boundary_attachment)

* `transcribe`
  * [x] [aws_transcribe_language_model](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_language_model)
  * [x] [aws_transcribe_medical_vocabulary](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_medical_vocabulary)
  * [x] [aws_transcribe_vocabulary](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary)
  * [x] [aws_transcribe_vocabulary_filter](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transcribe_vocabulary_filter)

* `transfer`
  * [x] [aws_transfer_tag](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/transfer_tag)

* `vpc_network`
  * [x] [aws_vpc_network_performance_metric_subscription](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/vpc_network_performance_metric_subscription)

Fixes #474 

I have:

- [x] Run `make reviewable test` to ensure this PR is ready for review.

### CODE REVIEW

File: config/externalnamenottested.go [hunk 1]
```diff
@@ -952,4 +952,121 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
...
+	// SageMaker Spaces can be imported using the id
+	// Example: arn:aws:sagemaker:us-west-2:123456789012:space/domain-id/space-name
+	"aws_sagemaker_space": config.IdentifierFromProvider,
+
+	// scheduler
+	//
+	// Schedules can be imported using the combination group_name/name
+	"aws_scheduler_schedule": config.IdentifierFromProvider,
+	// Schedule groups can be imported using the name
+	"aws_scheduler_schedule_group": config.IdentifierFromProvider,
```
Comment (turkenf) [inline config/externalnamenottested.go:1002]: It seems that we can use the following configuration, what do you think?

```suggestion
	"aws_scheduler_schedule_group": config. NameAsIdentifier,
```

**Review by turkenf — COMMENTED** [review]
Thanks @dverveiko for your PR, I left two comments

File: config/externalnamenottested.go [hunk 2]
```diff
@@ -952,4 +952,121 @@ var ExternalNameNotTestedConfigs = map[string]config.ExternalName{
...
+	// S3 Storage Lens configurations can be imported using the account_id and config_id, separated by a colon (:)
+	"aws_s3control_storage_lens_configuration": config.IdentifierFromProvider,
+
+	// sagemaker
+	//
+	// Models can be imported using the id
+	"aws_sagemaker_servicecatalog_portfolio_status": config.IdentifierFromProvider,
+	// SageMaker Spaces can be imported using the id
+	// Example: arn:aws:sagemaker:us-west-2:123456789012:space/domain-id/space-name
+	"aws_sagemaker_space": config.IdentifierFromProvider,
```
[the remark this answers is not in the document]
Comment (dverveiko) [reply config/externalnamenottested.go]: Fixed

File: config/externalnamenottested.go (hunk 1 above)
Comment (dverveiko) [reply config/externalnamenottested.go]: Fixed

**Review by turkenf — APPROVED** [review]
Thank you @dverveiko

**Pull request merged**